wordpress-mobile/WordPress-Android
# Convert photo picker fragment to kotlin to avoid memory leaks

Keeping views as fields leads to memory leaks. This PR fixes this issue by converting this the PhotoPickerFragment to Kotlin and replacing the fields with Kotlin synthetic accessors. As an additional advantage it will be much easier to use MVVM while working on the media picker consolidation project.
The changes should be straightforward without changing any logic.
Note: The code contains a solution to keep position in the list on orientation changes. This doesn't seem to work. I've checked the play store version and it doesn't work there either so I guess we'll fix this in the next phase of this project.

To test:
- Try adding a video/image block in GB editor
- Click on "Choose from device"
- Check that everything works as expected 

PR submission checklist:
